modules/serverless-autoscaling-maxscale-kn.adoc

```diff
@@ -4,26 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="serverless-autoscaling-maxscale-kn_{context}"]
-= Setting the maxScale annotation by using the Knative CLI
+= Setting the max-scale annotation by using the Knative CLI
 
-Using the `kn` CLI to set the `maxScale` annotation provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn service` command with the `--max-scale` flag to create or modify the `--max-scale` value for a service.
+Using the Knative (`kn`) CLI to set the `max-scale` annotation provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn service` command with the `--scale-max` flag to create or modify the `max-scale` value for a service.
 
 .Prerequisites
 
 * Knative Serving is installed on the cluster.
-* You have installed the `kn` CLI.
+* You have installed the Knative (`kn`) CLI.
 
 .Procedure
 
-* Set the maximum number of replicas for the service by using the `--max-scale` flag:
+* Set the maximum number of replicas for the service by using the `--scale-max` flag:
 +
 [source,terminal]
 ----
-$ kn service create <service_name> --image <image_uri> --max-scale <integer>
```

Comment:
For `kn` the `--max-scale` options work as well.

Maybe better to align docs to this for consistency.